sdk/tests/deqp/functional/gles3/es3fFboMultisampleTests.js

```diff
@@ -165,7 +165,7 @@ var DE_ASSERT = function(x) {
 
         // Render random-colored quads.
         /** @const {number} */ var numQuads = 8;
-        /** @type {deRandom.Random} */ var rnd = new deRandom.Random(9);
+        /** @type {deRandom.Random} */ var rnd = new deRandom.Random(7);
```

Comment:
Could you please add a comment here stating what you've mentioned in the commit message? Something like:

// The choice of random seed affects the correctness of the tests,
// because there are some boundary conditions which aren't handled
// correctly even in the C++ dEQP tests.

Thanks.